python/ray/tests/test_metrics.py

```diff
@@ -41,7 +41,11 @@ def test_worker_stats(ray_start_regular):
             continue
 
         # Check that the rest of the processes are workers, 1 for each CPU.
+        print(reply)
         assert len(reply.workers_stats) == num_cpus + 1
+        views = [view.view_name for view in reply.view_data]
+        assert "redis_latency" in views
```

Comment:
Should these be imported from somewhere as constants?

Reply:
There's a list at ray/stats/metric_defs.h, but here I'm just checking a random one exists for sanity.